models/response.js

```diff
@@ -0,0 +1,18 @@
+var Sequelize = require('sequelize');
+var sequelizeConnection = require('../db');
+
+var Question = require('./question')
+
+var Response = sequelizeConnection.define('response', {
+  choice: {
+    type: Sequelize.STRING
+  },
+  questionId: {
+    type: Sequelize.INTEGER
+  }
```

Comment:
no need to put questionId directly on schema. the association will automatically do this